x-pack/plugins/canvas/public/components/font_picker/font_picker.tsx

```diff
@@ -10,8 +10,13 @@ import PropTypes from 'prop-types';
 import React, { FunctionComponent } from 'react';
 import { fonts, FontValue } from '../../../common/lib/fonts';
 
+interface DisplayedFont {
```

Comment:
Can you use the `Font` interface defined in `/common/lib/fonts` instead of `DisplayedFont`? We already have types defined for the font object.

Reply:
Actually, nevermind. That's not actually what we want, since we allow the user to define their own fonts here.

Reply:
No changes necessary on the Canvas side 👍

Reply:
Thanks!